src/components/test-shelf.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+
+const Item = ({data}) => {
+  return (
+    <div className="coffee-list">
+      <p>{data.coffeeName} by {data.roasterName}</p>
+      <p className="coffee-size">{data.coffeeSize}</p>
+      <p>Roasted on {data.roastDate} </p>
```

Comment:
i'll just say it now but don't worry about having it here, but the next step here for this line will be to `yarn add moment` and then learn just enough moment.js to format `data.roastDate` in an interesting way (x days ago, last week, this week, etc)